From: Andrii Nakryiko <andriin@fb.com>
Date: Wed, 18 Dec 2019 13:43:14 -0800
Subject: bpftool: Simplify format string to not use positional args
Patch-mainline: v5.6-rc1
Git-commit: 7c43e0d6a526e7734eb854fe242886f52ccd06ac
References: bsc#1177028

Change format string referring to just single argument out of two available.
Some versions of libc can reject such format string.

Reported-by: Nikita Shirokov <tehnerd@tehnerd.com>
Signed-off-by: Andrii Nakryiko <andriin@fb.com>
Signed-off-by: Alexei Starovoitov <ast@kernel.org>
Acked-by: Yonghong Song <yhs@fb.com>
Link: https://lore.kernel.org/bpf/20191218214314.2403729-1-andriin@fb.com
Acked-by: Gary Lin <glin@suse.com>
---
 tools/bpf/bpftool/gen.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/tools/bpf/bpftool/gen.c
+++ b/tools/bpf/bpftool/gen.c
@@ -567,9 +567,9 @@ static int do_skeleton(int argc, char **
 			return -1;					    \n\
 		}							    \n\
 									    \n\
-		#endif /* %2$s */					    \n\
+		#endif /* %s */						    \n\
 		",
-		obj_name, header_guard);
+		header_guard);
 	err = 0;
 out:
 	bpf_object__close(obj);
